lib/courses/widgets/course_detail_screen.dart

```diff
@@ -35,55 +44,51 @@ class CourseDetailScreen extends StatelessWidget {
               onPressed: () => _showCourseFiles(context, course),
             ),
           ],
-          child: _LessonList(course: course),
+          child: Consumer<Bloc>(
+            builder: (context, bloc, _) {
+              return CachedBuilder(
```

Comment:
According to the [Effective Dart guide, we shouldn't do that](https://dart-lang.github.io/linter/lints/prefer_expression_function_bodies.html), because our declaration is more than a couple of lines and contains deeply nested expression.